Mayhemfile

```diff
@@ -0,0 +1,15 @@
+# Namespaced project name that the target belongs to
```

Comment:
Please omit duration and image name, and use generic name for the project field (omit your username `bmwtsn098`).